vizro-core/changelog.d/20240514_213231_antony.milne_dynamic_data_load_with_args.md

```diff
@@ -4,12 +4,10 @@ A new scriv changelog fragment.
 Uncomment the section that is right (remove the HTML comment wrapper).
 -->
 
-<!--
 ### Highlights ✨
 
-- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+- Enable dynamic data parametrization, so that different data can be loaded while the dashboard is running ([#482](https://github.com/mckinsey/vizro/pull/482))
```

Comment:
Just to say that you sent me down a rabbit hole https://english.stackexchange.com/questions/79847/parametrise-or-parameterise-a-curve